packages/vite-plugin-svelte/src/index.ts

```diff
@@ -96,7 +96,8 @@ export function svelte(inlineOptions?: Partial<Options>): Plugin {
 		},
 
 		async resolveId(importee, importer, customOptions, ssr) {
-			const svelteRequest = requestParser(importee, !!ssr);
+			const isSSR = !!(ssr || options.isSSRBuild);
```

Comment:
Is there a case where `ssr` is false, and `options.isSSRBuild` is true? Vite seems to has `ssr === resolvedConfig.build.ssr`, which is part to the derived value for `options.isSSRBuild`.

[ssr variable derived from here](https://github.com/vitejs/vite/blob/c08069cb3bdf7e7aa16d9cfaef4c1aea3a29cfa2/packages/vite/src/node/build.ts#L327-L329)

[if ssr, inject resolvedId forth parameter `ssr`](https://github.com/vitejs/vite/blob/c08069cb3bdf7e7aa16d9cfaef4c1aea3a29cfa2/packages/vite/src/node/build.ts#L358)

Reply:
you are right, i got confused because "svelte" never showed up as importee during ssr build and thought maybe vite had a problem with that flag.